Jieun/services/likeService.js

```diff
@@ -0,0 +1,11 @@
+const likeDao = require("../models/likeDao");
+
+const createLikes = async (userId, postId) => {
+  const likes = await likeDao.createLikes(userId, postId);
+
+  return likes;
```

Comment:
변수선언 및 비동기 처리 없이 곧바로 return하는게 깔끔하겠네요!
```suggestion
  return likeDao.createLikes(userId, postId);
```